### PR TITLE
move kubectl command to /docs/reference/generated directory 

### PR DESCRIPTION
**Pull request by wackxu**
move kubectl command to /docs/reference/generated directory 
/assign @steveperry-53

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6324)

### CODE REVIEW

File: _data/reference.yml [hunk 1]
```diff
@@ -49,20 +49,20 @@ toc:
 
 - title: kubectl CLI
   section:
-  - docs/user-guide/kubectl-overview.md
-  - docs/user-guide/kubectl/index.md
+  - docs/reference/generated/kubectl-overview.md
```
Comment (tengqm) [inline _data/reference.yml]: I think this is not generated.

Comment (wackxu) [reply _data/reference.yml]: @tengqm Yeah, I will split it into serveral commits. https://github.com/kubernetes/website/issues/6037 said that the /docs/user-guide/ directory will be  deprecated but I do not know where put these file. May be we should create a directory somewhere.

**Comment by wackxu** [comment]
close this to split into several PR

**Comment by steveperry-53** [comment]
@wackxu I think it will be best if I do the kubectl binary and kubectl commands. The kubectl commands are generated by [kubernetes-incubator/reference-docs](https://github.com/kubernetes-incubator/reference-docs/tree/master/gen-kubectldocs), and they are published a s single large file [here](https://kubernetes.io/docs/user-guide/kubectl/v1.8/).

**Comment by wackxu** [comment]
@steveperry-53 yeah, that is best you do it because I do not know these steps. If your need some help, you can call me, thanks.

**Pull request closed without merging**